### ISSUE TITLE
Allow server creation with no SSH key pair

### ISSUE THREAD

**Issue by j-rewerts** [browsh-org/baas#2]
I may be out to lunch on this, but [it looks like](https://github.com/browsh-org/baas/blob/8b4ffd37eabdc397bf76f710ad73cee3f211d508/ssh-server/ssh-server.go#L100) we currently require an SSH keypair to start our ssh-server. Guessing we mount the key into our container on startup? In Browsh's issue [#188](https://github.com/browsh-org/browsh/issues/188), it could help this user to simply be able to run the image with no keypair setup. Is this possible? It looks like gliderlabs SSH library supports it.

**Comment by tombh** [comment]
Yes, this is exactly right. The only reason that I mount keys is so that users don't get that "unkown host" warning every time Kubernetes changes the SSH server's Docker container. So it should be easy to add a flag to simply not use the `hostKeyFilePath` arg.

(BTW, I've actually got my original Browsh dev system back up and running), so if I get some time I can actually make pull requests again :)
